packages/blade/src/components/Input/OTPInput/OTPInput.tsx

```diff
@@ -243,6 +243,7 @@ const OTPInput = ({
             errorText={errorText}
             helpText={helpText}
             hideFormHint={true}
+            type="numeric"
```

Comment:
How does this affect the UI? Can you post some screenshots please? Does it also show the number increase number decrease controls?